### PR TITLE
OTJ 16 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/caught_in_the_crossfire.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Name:Caught in the Crossfire
+ManaCost:R R
+Types:Instant
+A:SP$ Charm | Choices$ DBDamageOutlaw,DBDamageNonOutlaw | MinCharmNum$ 1 | CharmNum$ 2 | Spree$ True
+SVar:DBDamageOutlaw:DB$ DamageAll | SpreeCost$ 1 | NumDmg$ 2 | ValidCards$ Creature.Outlaw | SpellDescription$ CARDNAME deals 2 damage to each outlaw creature. (Assassins, Mercenaries, Pirates, Rogues, and Warlocks are outlaws.)
+SVar:DBDamageNonOutlaw:DB$ DamageAll | SpreeCost$ 1 | NumDmg$ 2 | ValidCards$ Creature.nonOutlaw | SpellDescription$ CARDNAME deals 2 damage to each non-outlaw creature.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/caught_in_the_crossfire.txt:6]: wrong valid since it's not real type

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/caught_in_the_crossfire.txt:6]: Should it be Creature.!Outlaw

Comment (dracontes) [reply forge-gui/res/cardsfolder/upcoming/caught_in_the_crossfire.txt:6]: Tested it and `Creature.!Outlaw` works. Also you need `K:Spree` at the top (under the type line) so the additional costs for the choices are actually added.

File: forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Breeches, the Blastmaker
+ManaCost:1 U R
+Types:Legendary Creature Goblin Pirate
+PT:3/3
+K:Menace
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl | TriggerZones$ Battlefield | Execute$ TrigFlip | ValidActivatingPlayer$ You | ActivatorThisTurnCast$ EQ2 | TriggerDescription$ Whenever you cast your second spell each turn, you may sacrifice an artifact. If you do, flip a coin. When you win the flip, copy that spell. You may choose new targets for the copy. When you lose the flip,CARDNAME deals damage equal to that spell's mana value to any target.
+SVar:TrigFlip:AB$ FlipACoin | Cost$ Sac<1/Artifact> | WinSubAbility$ DBCopy | LoseSubAbility$ DBDamage
+SVar:DBCopy:DB$ CopySpellAbility | Defined$ TriggeredSpellAbility | MayChooseTarget$ True | SpellDescription$ When you win the flip, copy that spell. You may choose new targets for the copy.
+SVar:DBDamage:DB$ DealDamage | ValidTgts$ Any | NumDmg$ X | SpellDescription$ When you lose the flip, CARDNAME deals damage equal to that spell's mana value to any target.
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt:9]: I think this needs to be wrapped in an immediatetrigger ?

File: forge-gui/res/cardsfolder/upcoming/boom_box.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Boom Box
+ManaCost:2
+Types:Artifact
+A:AB$ Destroy | Cost$ 6 T Sac<1/CARDNAME> | ValidTgts$ Artifact | TgtPrompt$ Select up to one target artifact | TargetMin$ 0 | TargetMax$ 1 | SubAbility$ DBDestroyCreature | SpellDescription$ Destroy up to one target artifact, up to one target creature, and up to one target land.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/boom_box.txt:4]: check _Decimate_ on how to script this correctly

File: forge-gui/res/cardsfolder/upcoming/bounding_felidar.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Name:Bounding Felidar
+ManaCost:5 W
+Types:Creature Cat Beast Mount
+PT:4/7
+T:Mode$ Attacks | ValidCard$ Card.Self+IsSaddled | TriggerZones$ Battlefield | Execute$ TrigPutCounterAll | TriggerDescription$ Whenever CARDNAME attacks while saddled, put a +1/+1 counter on each other creature you control. You gain 1 life for each of those creatures.
+SVar:TrigPutCounterAll:DB$ PutCounterAll | ValidCards$ Creature.YouCtrl+Other | CounterType$ P1P1 | CounterNum$ 1 | SubAbility$ DBGainLife
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bounding_felidar.txt:6]: StrictlyOther

File: forge-gui/res/cardsfolder/upcoming/double_down.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
+Name:Double Down
+ManaCost:3 U
+Types:Enchantment
+T:Mode$ SpellCast | ValidCard$ Card.Outlaw | ValidActivatingPlayer$ You | Execute$ TrigCopy | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast an outlaw spell, copy that spell. (Assassins, Mercenaries, Pirates, Rogues, and Warlocks are outlaws. Copies of permanent spells become tokens.)
+SVar:TrigCopy:DB$ CopySpellAbility | Defined$ TriggeredSpellAbility | AILogic$ Always | MayChooseTarget$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/double_down.txt:5]: two weird params

File: forge-gui/res/cardsfolder/upcoming/three_steps_ahead.txt [hunk 6]
```diff
@@ -0,0 +1,11 @@
+Name:Three Steps Ahead
+ManaCost:U
+Types:Instant
+K:Spree
+A:SP$ Charm | Choices$ DBCounter,DBCopy,DBLoot | MinCharmNum$ 1 | CharmNum$ 3 | Spree$ True
+SVar:DBCounter:DB$ Counter |  SpreeCost$ 1 B | TargetType$ Spell | ValidTgts$ Card | SpellDescription$ Counter target spell.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/three_steps_ahead.txt]: wrong cost

File: forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt [hunk 7]
```diff
@@ -0,0 +1,11 @@
+Name:Breeches, the Blastmaker
+ManaCost:1 U R
+Types:Legendary Creature Goblin Pirate
+PT:3/3
+K:Menace
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl | TriggerZones$ Battlefield | Execute$ TrigFlip | ValidActivatingPlayer$ You | ActivatorThisTurnCast$ EQ2 | TriggerDescription$ Whenever you cast your second spell each turn, you may sacrifice an artifact. If you do, flip a coin. When you win the flip, copy that spell. You may choose new targets for the copy. When you lose the flip,CARDNAME deals damage equal to that spell's mana value to any target.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt:6]: spacing

File: forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt (hunk 2 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt:9]: yes

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt:9]: Same with the one that copies the spell ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt:9]: yes

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt]: Hmm I can't seem to find the correct way to remember the triggeredobject on the copy

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt]: Use Spawner Syntax instead

File: forge-gui/res/cardsfolder/upcoming/bounding_felidar.txt (hunk 4 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/bounding_felidar.txt]: maybe update both places...?

File: forge-gui/res/cardsfolder/upcoming/boom_box.txt (hunk 3 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/boom_box.txt]: any specific reason you left out the AILogic on the first one...?

File: forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt (hunk 2 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt]: I don't understand why you switched the triggeringObject to Card

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt]: What should the correct syntax be in this case ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt]: you want to copy a spell and not a card, so TriggeredSpellAbility like always

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt]: Yeah I've tried this before but with no succes

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt]: now use it for both

File: forge-gui/res/cardsfolder/upcoming/bucolic_ranch.txt [hunk 8]
```diff
@@ -0,0 +1,11 @@
+Name:Bucolic Ranch
+ManaCost:no cost
+Types:Land Desert
+A:AB$ Mana | Cost$ T | Produced$ C | SpellDescription$ Add {C}.
+A:AB$ Mana | Cost$ T | Produced$ Any | Amount$ 1 | RestrictValid$ Spell.Mount | SpellDescription$ Add one mana of any color. Spend this mana only to cast a Mount spell.
+A:AB$ PeekAndReveal | Cost$ 3 T | PeekAmount$ 1 | RevealValid$ Card.Mount | RevealOptional$ True | RememberRevealed$ True | SubAbility$ DBChangeZone | SpellDescription$ Look at the top card of your library. If it's a Mount card, you may reveal it and put it into your hand. If you don’t put it into your hand, you may put it on the bottom of your library.
+SVar:DBChangeZone:DB$ ChangeZone | Defined$ Remembered | Origin$ Library | Destination$ Hand | SubAbility$ DBChangeZone2
+SVar:DBChangeZone2:DB$ ChangeZone | Optional$ True | Defined$ TopOfLibrary | Origin$ Library | Destination$ Library | LibraryPosition$ -1 | ConditionDefined$ Remembered | ConditionPresent$ Card | ConditionCompare$ EQ0 | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bucolic_ranch.txt:8]: don't you need to forget for the condition to work?

File: forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt (hunk 2 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt]: I see it doesn't work fully yet, will make a fix

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/breeches_the_blastmaker.txt]: Thank you !

**Review by tool4ever — APPROVED** [review]

**Pull request merged**